libraries/logging/include/logging_stack.h

```diff
@@ -33,30 +33,50 @@
 
 /* FreeRTOS Include. */
 #include "FreeRTOS.h"
+#include "iot_logging_task.h"
 
 /* Standard Include. */
 #include <stdint.h>
 #include <string.h>
 
-/* Metadata information to prepend to every log message. */
-
-/* Macro to extract only the file name from file path to use for metadata in
- * log messages. */
-#ifdef _MSC_VER
-    #define FILENAME           ( strrchr( __FILE__, '\\' ) ? strrchr( __FILE__, '\\' ) + 1 : __FILE__ )
-#else
-    #define FILENAME           ( strrchr( __FILE__, '/' ) ? strrchr( __FILE__, '/' ) + 1 : __FILE__ )
+/**
+ * @brief This config enables extra verbosity in log messages with metadata information
+ * about the source library and location of the log message.
+ * This requires that the toolchain support C99 (for variadic macros) and the GNU
+ * extension for comma elision in variadic macros (with ##__VA_ARGS__).
+ * If this flag is enabled, you can change the metadata information from the logging_stack.h
+ * file.
+ *
+ * @note By default, this configuration is disabled by
+ */
+#ifndef LOGGING_ENABLE_METADATA_WITH_C99_AND_GNU_EXTENSION
+    #define LOGGING_ENABLE_METADATA_WITH_C99_AND_GNU_EXTENSION    0
```

Comment:
Reading the README I got the impression that the default is `1`.

Reply:
Nope, will make it clear in the README